### ISSUE TITLE
Exclude files in c# project from the content directory

### ISSUE THREAD

**Issue by PvdWaltDevRoc** [NuGet/Home#8786]
Nuget.exe Version: 4.9.4.5839

I have no idea why I'm struggling with this, it should be so easy?

My c# project have an additional folder with external dependent dlls. This is done to keep the Dlls with the project.

When I pack my Nuget package from the c# project file it includes these dlls in the content folder however I don't want them there and I have tried to exclude them in any way possible except removing them from the project that obviously works.

This is my pack output

Id: TPNApplicationBlock
Version: 2019.10.11
Authors: TPN Credit Bureau
Description: TPN Application Block
License Url: https://www.tpn.co.za/
Tags: TPN, Application, Block
Dependencies:

Added file '[Content_Types].xml'.
Added file '_rels/.rels'.
Added file 'content/Assemblies/Microsoft.Practices.EnterpriseLibrary.Common.dll'.
Added file 'content/Assemblies/Microsoft.Practices.EnterpriseLibrary.Data.dll'.
Added file 'content/Assemblies/Microsoft.Practices.ServiceLocation.dll'.
Added file 'content/Assemblies/Microsoft.Practices.Unity.dll'.
Added file 'content/Assemblies/Microsoft.Practices.Unity.Interception.dll'.
Added file 'lib/net462/Microsoft.Practices.EnterpriseLibrary.Common.dll'.
Added file 'lib/net462/Microsoft.Practices.EnterpriseLibrary.Data.dll'.
Added file 'lib/net462/Microsoft.Practices.ServiceLocation.dll'.
Added file 'lib/net462/Microsoft.Practices.Unity.dll'.
Added file 'lib/net462/Microsoft.Practices.Unity.Interception.dll'.
Added file 'lib/net462/TPNApplicationBlock.dll'.
Added file 'lib/net462/TPNExtensions.dll'.
Added file 'package/services/metadata/core-properties/12e5b3cd4a34494da70350737183ea5c.psmdcp'.
Added file 'TPNApplicationBlock.nuspec'.

Successfully created package 'D:\Files\Source\Common\TPN_ApplicationBlock\Solution_Trunk\TPNApplicationBlock\TPNApplicationBlock.2019.10.11.nupkg'.

I want to remove these:
Added file 'content/Assemblies/Microsoft.Practices.EnterpriseLibrary.Common.dll'.
Added file 'content/Assemblies/Microsoft.Practices.EnterpriseLibrary.Data.dll'.
Added file 'content/Assemblies/Microsoft.Practices.ServiceLocation.dll'.
Added file 'content/Assemblies/Microsoft.Practices.Unity.dll'.
Added file 'content/Assemblies/Microsoft.Practices.Unity.Interception.dll'.

In summary. All dlls in my Assemblies folder should go to the lib folder which succeed and I have no problem with it but they shouldn't go to contents because that creates a folder in the project if I reference the nuget package
 with the dependent dlls which is just ugly!

Tried to upload sample files but it never completes so I gave up...

**Comment by nkolev92** [comment]
Hey @rrelyea, since you are in hot seat, can you please revisit this issue and help with the customer's question.

**Comment by zivkan** [comment]
I can't guess how this project was being packed, so I can't give advice on how to solve the problem.

My first feeling is that this project was packed with our MSBuild task (so either `dotnet pack` or `msbuild -t:pack`), in which case my gut reaction is that the `*.dll` files were MSBuild `Content` items with `pack="true"`metadata. However, if that were the case I would also expect the files to be in `contentFIles/`. Since it's not in both `content/` and `contentFiles/`, this makes me think this is not packed with our MSBuild tasks, and therefore this was probably packed with `nuget.exe pack`.

@PvdWaltDevRoc I know this is a very old issue, and you said you tried to upload a sample but it was timing out, but can you give us more details about the project and how it was packed?  Did you write your own nuspec, or did you let NuGet independently pack the project? How exactly were the external dependent dlls added to the project, such that NuGet packs them when you pack the project?

**Comment by kartheekp-ms** [comment]
@PvdWaltDevRoc - Please provide feedback to [above comment](https://github.com/NuGet/Home/issues/8786#issuecomment-632462320) so that we can understand the issue better.

> Tried to upload sample files but it never completes so I gave up...

Can you please try uploading the same files again?
